scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -18,8 +18,9 @@ export CAPM3_REL_TO_VERSION
 export CAPIRELEASE="v0.3.15"
 export CAPI_REL_TO_VERSION="v0.3.16"
 
-export KUBERNETES_VERSION="v1.21.1"
-export UPGRADED_K8S_VERSION="v1.21.2"
+export FROM_K8S_VERSION="v1.21.1"
+export KUBERNETES_VERSION=${FROM_K8S_VERSION}
+export UPGRADED_K8S_VERSION="v1.21.3"
```

Comment:
Just keep in mind that if you would like to switch to 1.21.3, node images with 1.21.3 will need to be built as well from airship-dev-tools.